modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/Blat.pm

```diff
@@ -108,36 +110,15 @@ sub configure_runnable {
   #my $oocFile = $dbChunkFile;
   #$oocFile =~ s/(.fa)/\/5ooc/;
 
-  #my $options = $self->options . " -ooc=$oocFile";
-
-  my $mlss = $self->compara_dba->get_MethodLinkSpeciesSetAdaptor->fetch_by_dbID($self->param('mlss_id'));
-  my $options = $mlss->get_value_for_tag("param");
-
-  #If not in method_link_species_set_tag table (new pipeline) try param (old pipeline)
-  if (!$options) {
-      my $option_str = "options_" . $self->param('mlss_id');
-      $options = $self->param($option_str);
-  }
-
-  #Check options have been set.
-  throw("Unable to find options in method_link_species_set_tag table or in $self->param('options') ") unless (defined $options);
-
-  if($self->debug) {
-    print("running with analysis '".$self->input_job->analysis->logic_name."'\n");
-    print("  options : ", $options, "\n");
-  }
+  $self->compara_dba->dbc->disconnect_if_idle();
 
-  $self->param('runnable', []);
-  my $runnable = Bio::EnsEMBL::Compara::Production::Analysis::Blat->
-    new(
-	-query      => $qyChunkFile,
-	-database   => $dbChunkFile,
-	-options    => $options,
-       );
-  push @{$self->param('runnable')}, $runnable;
+  my $starttime = time();
+  my $output = Bio::EnsEMBL::Compara::Production::Analysis::Blat::run_blat($self, $qyChunkFile, $dbChunkFile);
```

Comment:
Why do we need to call it like this instead of just `$self->run_blat($qyChunkFile, $dbChunkFile);` ?